### PR TITLE
adding JavaScript Regex Panels

### PR DESCRIPTION
**Pull request by vijeysrc**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [ ] you only add one (!) new helper per pull request.
- [ ] you have checked if an open PR already exists.
- [ ] the submitted website is focused on a single, development related issue.
- [ ] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

**Comment by vijeysrc** [comment]
Hi @stefanjudis, Please let me know if you see any issues or pending tasks with this PR.

Thanks,
Vijey

File: helpers/javascript-regex-panels.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "JavaScript Regex Panels",
+  "desc": "To practice, perfect, or work on your JavaScript Regular Expressions ",
```
Comment (stefanjudis) [inline helpers/javascript-regex-panels.json:3]:
```suggestion
  "desc": "Test JavaScript Regular Expressions ",
```

File: helpers/javascript-regex-panels.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "url": "https://www.regexp.in",
```
Comment (stefanjudis) [inline helpers/javascript-regex-panels.json:4]:
```suggestion
  "url": "https://www.regexp.in/practice-area.html",
```

**Comment by stefanjudis** [comment]
Hey @vijeysrc, thank you so much for your contribution. :) Sorry for the delay. Life's busy right now. 🙈

I added some suggestions to only link the regex playground. Let me know if that works for you. :)

**Comment by vijeysrc** [comment]
> Hey @vijeysrc, thank you so much for your contribution. :) Sorry for the delay. Life's busy right now. 🙈
> 
> I added some suggestions to only link the regex playground. Let me know if that works for you. :)

No worries.  I have committed / applied your suggestions. 

Thanks a lot for having this platform where we can share our tools.

**Comment by stefanjudis** [comment]
thank you! 🙇‍♂️

**Pull request merged**